### DISCUSSION TITLE
0.3.0 beta

### DISCUSSION THREAD

**Discussion by rtertiaer** [micro-nova/AmpliPi#515] [General]
This is the first beta release of our 0.3.0 series. It lands a couple of hefty improvements:
* A brand new UI, written in React :sunglasses: 
* Bluetooth support; simply plug an adapter into your AmpliPi, reboot your appliance and pair with it (note: only one concurrent Bluetooth stream is supported at this time) Thanks again @kjk2010!
* The updates to spotifyd mentioned in [a prior prerelease](https://github.com/micro-nova/AmpliPi/releases/tag/spotifyd-update-prerelease)
* Plus errata & bugfixes.

We would love your feedback on this release! If you encounter any problems with the new WebUI, please file an issue with expected behavior, actual behavior and notes on how to replicate it. We appreciate it :)

<hr /><em>This discussion was created from the release <a href='https://github.com/micro-nova/AmpliPi/releases/tag/0.3.0-beta1'>0.3.0 beta</a>.</em>

**Comment by kaikreuzer** [comment]
Hi! I have updated to 0.3.0 beta 2, but it seems I am having some issues - the PA feature (with openHAB integration) isn't working anymore and activating/deactivating zones isn't reliable anymore. I thus tried to downgrade to 0.2.1 again, but when going to the "Updates" menu item, the browser is redirected to `http://amplipi:5001/update`, which does not exist (port 5001 is not served). Checking on "About", I am told I am currently running version `0.2.1+731258c-Webapp2-RUM-dirty` - is this correct or did the update to 0.3.0 beta 2 didn't work as expected? What is the simplest way to downgrade to 0.2.1 again?

**Reply by rtertiaer** [reply]
Howdy Kai! Thanks so much for your feedback on our beta :) We really appreciate your willingness to try it out and report back.

If you haven't yet, can you try rebooting the appliance to restart the `amplipi-updater` service? It ought to be listening on `http://amplipi.local:5001/update` at all times. If it still fails to launch after a reboot, can you please send the logs (found at Settings -> About -> Logs in the new UI) to us at support@micro-nova.com ?

The version tag in Beta 2 is a bug, but does reflect that it was updated; the latest beta, released today, correctly reflects its version, 0.3.0-beta.3. We also fixed some bugs with the API that were surfaced when we fixed our test suite in this latest beta. We'd love to know if the issues you are having are repeatable there. In our testing of the PA functionality in 0.3.0-beta.3, we were not able to repeat any failure; if it continues, can you share with us the API call that fails?
